### PR TITLE
Documentation language changes

### PR DESCRIPTION
**Pull request by harshitmalik03**

### CODE REVIEW

File: dev/source/docs/how-the-team-works-voting.rst [hunk 1]
```diff
@@ -42,9 +42,9 @@ Description belows how to creat a form from scratch, alternatively you can use `
   - set the "Email To" field to "ardupilot-dev-team@googlegroups.com"
   - fill in the Subject and Message fields
   - to allow others to see the responses (including ardupilot.devel@gmail.com), click the "Collaborators" link and add their email in the "Invite people" field.  If you are submitting the vote as yourself, include 
-  - check "Include form in email" and press "Send"
+  - check "Include form in email" and press "Send".
```
Comment (Hwurzburg) [inline dev/source/docs/how-the-team-works-voting.rst:45]: Not really necessary, but the line above it really does not make sense....I know its existing, but @rmackay9 , what are you supposed to include? if its a vote for yourself?

**Comment by harshitmalik03** [comment]
Hey,
Im an undergraduate student from India and I can code in java and python
and can also do basic documentation stuff.
Can you please help me out on how do i start with contributing to open
source projects ! It would be nice of you if you can help me out via
call.(Im available on discord too) .
Thank you so much

File: dev/source/docs/how-the-team-works-voting.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/how-the-team-works-voting.rst:45]: @rmackay9 bump

**Comment by Hwurzburg** [comment]
@harshitmalik03 welcome to ArduPilot! if you want to contribute to the wiki, read https://ardupilot.org/dev/docs/common-wiki_editing_guide.html, for code: https://ardupilot.org/dev/docs/building-the-code.html and https://ardupilot.org/dev/docs/contributing.html

**Comment by harshitmalik03** [comment]
Thanks a lot. Can you tag the beginner stuff? Any code in java or
documentation issues which I can try.

**Comment by Hwurzburg** [comment]
@rmackay9 still need answer to my question...thanks

File: dev/source/docs/how-the-team-works-voting.rst (hunk 1 above)
Comment (rmackay9) [reply dev/source/docs/how-the-team-works-voting.rst:45]: I think we should zap the end of the line, "If you are submitting the vote as yourself, include "

File: dev/source/docs/how-the-team-works-voting.rst [hunk 2]
```diff
@@ -42,9 +42,9 @@ Description belows how to creat a form from scratch, alternatively you can use `
...
 
-Normally the vote should be open for 1 week and be discussed at the weekly development call.
+The votes are generally open for 1 week and will be discussed at the weekly development call.
```
Comment (rmackay9) [inline dev/source/docs/how-the-team-works-voting.rst:47]: The original english is correct actually.  The replacement is also correct but the change doesn't improve the english I think.

File: dev/source/docs/how-the-team-works-voting.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/how-the-team-works-voting.rst:45]: zapped....by a direct commit

**Comment by Hwurzburg** [comment]
since I removed the dangling incomplete sentence already...and the rest are deemed as no real change....I am closing this...
but @harshitmalik03 , thanks for the contribution, and please continue to make them...

**Pull request closed without merging**